app/main/views/users.py

```diff
@@ -76,15 +82,18 @@ def list_users():
 
         user_query = user_query.filter(User.supplier_id == supplier_id)
 
-    users = user_query.paginate(
+    paginated_users = user_query.paginate(
         page=page,
         per_page=current_app.config['DM_API_SERVICES_PAGE_SIZE'],
     )
 
     return jsonify(
-        users=[user.serialize() for user in users.items],
+        users=[user.serialize() for user in paginated_users.items],
+        meta={
+            "total_pages": paginated_users.pages
```

Comment:
@TheDoubleK The link to the last page looks something like this `http://localhost:5000/users?page=25&only_buyers=True` so I guess I could extract the total pages from there. Is that better than just passing the number in the json?